deps/zlib/zlib.gyp

```diff
@@ -163,4 +163,4 @@
       ],
     }],
   ],
-}
+}
```

Comment:
Unnecessary change

Reply:
I'm OK with stripping out trailing spaces in this PR rather than leaving them or doing it in a separate PR.